pkgs/development/python-modules/kubernetes-asyncio/default.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  coreutils,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  aiohttp,
+  certifi,
+  python-dateutil,
+  pyyaml,
+  six,
+  urllib3,
+
+  # tests
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "kubernetes-asyncio";
+  version = "31.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tomplus";
+    repo = "kubernetes_asyncio";
+    rev = "refs/tags/${version}";
+    hash = "sha256-YKBqhUeLqLiQ6bK235zTm4salnSLUxl4DUiFLQSjWqw=";
+  };
+
+  postPatch = ''
+    substituteInPlace kubernetes_asyncio/config/google_auth_test.py \
+      --replace-fail "/bin/echo" "${lib.getExe' coreutils "echo"}"
+  '';
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    aiohttp
+    certifi
+    python-dateutil
+    pyyaml
+    six
+    urllib3
+  ];
+
+  pythonImportsCheck = [
+    "kubernetes_asyncio"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  __darwinAllowLocalNetworking = true;
```

Comment:
It looks like this is deprecated/renamed:
https://github.com/NixOS/nixpkgs/blob/7783a48f3858b2783be21167118735ae8407e064/pkgs/stdenv/generic/make-derivation.nix#L130-L137
but I'm not able to find any docs pointing to why or what to use instead.

Perhaps @NixOS/darwin-maintainers might be able to help us?

Reply:
Apparently, it will just be removed from the derivation arguments and we can still use it without any problem.